### PR TITLE
Source from GitHub tarball (from #105)

### PR DESCRIPTION
**Pull request by flferretti**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
Is this PR actually using go at all? I don't see anything in the logs.

**Comment by flferretti** [comment]
> Is this PR actually using go at all? I don't see anything in the logs.

I found out that it is not required for this W&B version, sorry. 

Ready for review

**Comment by flferretti** [comment]
> Shouldn't this stay noarch? Isn't any compilation coming in in the next version?

At this point yes, you're right!

**Comment by desilinguist** [comment]
Thank you so much for your work on this @flferretti 🙏🏽!  This will unblock a bunch of our open-source projects that rely on a conda package of W&B.

**Comment by desilinguist** [comment]
Can I please request the code owners to approve and merge this MR, assuming all the issues are resolved?

**Comment by flferretti** [comment]
@xhochy @synapticarbors friendly ping. Thanks!

**Review by timoffex — APPROVED** [review]
Hi @flferretti, I'm a maintainer of `wandb`, and I'm taking some time now to look into our conda situation :) Thanks a huge amount for your effort!

I'm still ramping up on how conda works and the discussions about `wandb`'s Go and Swift requirements (and recently Rust...), so I appreciate your patience. I wrote the build code (using `hatchling`) for `wandb`, and I hope I can transfer that knowledge to the conda package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,36 +7,38 @@ package:
...
-  script: {{ PYTHON }} -m pip install . -vv
+  script_env: WANDB_BUILD_SKIP_APPLE=true  # [osx]
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - python >=3.7
+    - hatchling
```
Comment (timoffex) [inline recipe/meta.yaml:28]: I believe the `hatchling` dependency was added in version 0.17, so it shouldn't be needed here. Am I wrong?

Comment (flferretti) [reply recipe/meta.yaml]: You're absolutely right! Addressed in f0ff1893b5c2809731d353526a04deb1c01dcbee

**Comment by timoffex** [comment]
I believe `setuptools` is correct for `0.16.6`.

**Pull request merged**